spacer/extract_features.py

```diff
@@ -132,7 +133,7 @@ def __call__(self, im, rowcols):
                         'batch_size': 10}
 
         # Crop patches
-        patch_list = crop_patch(im, rowcols, torch_params['crop_size'])
+        patch_list = crop_patches(im, rowcols, torch_params['crop_size'])
 
```

Comment:
also `del im` after cropping patches in `EfficientNetExtractor()`?

Reply:
Yeah, exactly, that helped also. I don't have a clear record of how much each helped, but allocating that `nbr_pointsx224x224x3` array of continuous memory was pretty expensive.

Reply:
Ah, sorry, I see what you mean now. You are right. 

Actually, the more I think about it, we crop first using Image.crop() from PIL and convert the patches to numpy after. That way we don't even have to keep the whole numpy array image in memory. I might try that instead.

Reply:
... I added that delete. Saving the other optimizations for another day ;)

Reply:
Great, I can take a look at that after and code some memory usage test, that's what I did in my codebase